src/main/kotlin/org/domnikl/algorithms/sorting/CountingSort.kt

```diff
@@ -0,0 +1,19 @@
+package org.domnikl.algorithms.sorting
+
+fun IntArray.countingSort(): IntArray {
+    val min = this.min()!!
+    val max = this.max()!!
+    val counts = IntArray(max - min + 1)
+
+    this.forEach { counts[it-min]++ }
+
+    var pointer = 0
+    counts.forEachIndexed { index, i ->
+        repeat(i){
```

Comment:
coding style: space between closing brace and opening curly brace

Reply:
you want to add ktlint?